docs/aws/deploy-heroku.md

```diff
@@ -0,0 +1,34 @@
+## How to setup a heroku app with cloudfront
+
+Assuming you have created your heroku app and have the link ready e.g. (appname.herokuapp.com)
+
+### Create Cloudfront Distribution
+
+1. Open cloudfront and hit **Create Distribution**
+2. In this case we are going to click **Get Started** on create web distribution.
+3. Fill in the blank for **Origin Settings section**.  
+    3.1 Origin Domain Name: This should be your heroku URL (appname.herokuapp.com)
+    3.2 Origin Path: This should be your base route for example: **appname.herokuapp.com/home** then this should be **/home**
+    3.3 Origin ID: This will be auto generated.
+    3.4 Origin Custom Headers: You don't need this unless you need to add header in every request.
```

Comment:
maybe something along the lines of: Use this to pass custom HTTP Headers from CloudFront to your origin server